cli/tests/integration/lsp_tests.rs

```diff
@@ -13,42 +12,13 @@ use std::collections::HashSet;
 use std::fs;
 use std::process::Stdio;
 use test_util::deno_cmd_with_deno_dir;
-use test_util::deno_exe_path;
 use test_util::env_vars_for_npm_tests;
-use test_util::http_server;
 use test_util::lsp::LspClient;
+use test_util::lsp::LspClientBuilder;
 use test_util::testdata_path;
-use test_util::TempDir;
+use test_util::TestContextBuilder;
 use tower_lsp::lsp_types as lsp;
 
-fn load_fixture(path: &str) -> Value {
-  load_fixture_as(path)
-}
```

Comment:
I inlined all the fixtures. Having them in separate files created too much of a disconnect and we can work on moving more of this to be abstracted away in the TestLspClient.

Reply:
Strongly agree, good change